### DISCUSSION TITLE
podman usage resulting in `Inconsistency detected by ld.so` errors

### DISCUSSION THREAD

**Discussion by manan-gup** [containers/podman#20814] [Q&A] [answered]
Hi, my machine lost power while a container was running on podman and now any podman operation like `podman images`, `podman container ls` or running a container result in this error:
```
Inconsistency detected by ld.so: ../sysdeps/x86_64/dl-machine.h: 487: elf_machine_rela_relative: Assertion `ELFW(R_TYPE) (reloc->r_info) == R_X86_64_RELATIVE' failed!
```
I tried doing `podman system reset` but that also resulted in the same error. Is there a way to manually reset the podman installation without uninstalling it from the system?

**Comment by edsantiago** [comment]
I vaguely recall seeing something similar, and fixing it with `dnf reinstall container-selinux`. If you're on an RPM-based distro with SELinux enabled, can you give that a try?

**Reply by manan-gup** [reply]
I'm on a silverblue style atomic distro. Any suggestions for that?

**Reply by edsantiago** [reply]
Not really, but you could try _temporarily_ `# setenforce 0` to see if the problem is SELinux-related. If the problem goes away with SELinux disabled, reenable (`# setenforce 1`) then try relabeling:
```console
# restorecon -v -r /var/lib/containers   (if root)
$ restorecon -v -r ~/.local/share/containers (rootless)
```
This is a long shot, though.

**Reply by manan-gup** [reply]
Ok, I'll give it a go. Thanks.

**Reply by manan-gup** [reply, marked as answer]
I'm not sure what did it but I did `setenforce 0` and then rebooted. Now its working even after re-enabling with `setenforce 1`.

**Reply by Rtoax** [reply]
> Ok, I'll give it a go. Thanks.

Hi, does it works for you?

**Reply by rhatdan** [reply]
Are you seeing SELinux issues?
